Agrega un método de login a servidor JWT scheduled

gabrielci opened the pull request:
Agrega método de login a servidor JWT que se corre cada cierta cantidad de tiempo, utilizando un URL de Login junto a un Usuario y Contreseña

Los 4 parametros mencionados se deben definir en el application.properties, se deja un ejemplo que detalla el nombre de los campos y valores utilizados para conectar a una instancia del Joko Backend Starter Kit (Con un puerto distinto al 8080).
```
scheduler.login.cron.timer=*/5 * * * * *
scheduler.login.url=http://localhost:8085/api/login
scheduler.login.username=admin
scheduler.login.password=123456
```

El timer seria un String de tiempo definido como en el crontab: https://crontab.guru/

alefq commented:
1. Estas propiedades nuevas mencionadas en el comentario, y en general; cualquier nueva propiedad que sea requerida, deben ser incluídas en los archivos de ejemplo `application.properties{.sample}` 
2. Utilizaremos los criterios de software factory de [Joko](https://joko.miraheze.org/wiki/Software_Factory). Para este ticket, aplicar la revisión de QA. Para adecuar el resto; crearé un nuevo ticket.

gabrielci committed:
Agrega método de login a servidor JWT que se corre cada cierta cantidad de tiempo que uno define

- Parametros configurables en el aplication.properties

- Se agrego un servicio de prueba para obtener el secreto guardado por el Worker

alefq commented:
Buen trabajo Gabriel. Pude levantar sin problemas el proyecto y hacer funcionar el login desde la tarea programada.
